src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -1,9 +1,42 @@
-<Project Sdk="Microsoft.Build.NoTargets">
+<Project Sdk="Microsoft.NET.SDK">
   <!-- These are wrapper project files for packaging.-->
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0</TargetFrameworks>
-    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
-    <!-- Avoid trying to binplace non existent documentation file.-->
-    <EnableBinPlacing>false</EnableBinPlacing>
+    <TargetFrameworks>$(NetCoreAppToolCurrent);net472</TargetFrameworks>
+    <PackageId>$(MSBuildProjectName)</PackageId>
+    <AvoidRestoreCycleOnSelfReference>true</AvoidRestoreCycleOnSelfReference>
+    <AssemblyName>Microsoft.NETCore.Platforms.BuildTasks</AssemblyName>
+
+    <IsSourceProject>false</IsSourceProject>
+    <IsPackable>true</IsPackable>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <NoWarn>$(NoWarn);NU5128</NoWarn> <!-- No Dependencies-->
+    <PackageDescription>Provides runtime information required to resolve target framework, platform, and runtime specific implementations of .NETCore packages.</PackageDescription>
   </PropertyGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
+    <Compile Include="BuildTask.Desktop.cs" />
+    <Compile Include="AssemblyResolver.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="BuildTask.cs" />
+    <Compile Include="Extensions.cs" />
+    <Compile Include="GenerateRuntimeGraph.cs" />
+    <Compile Include="RID.cs" />
+    <Compile Include="RuntimeGroup.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="runtime.json" PackagePath="/" />
+    <Content Include="_._" PackagePath="lib/netstandard1.0" />
```

Comment:
Why not move that "_._" file into a common location to be used by multiple packages?

Reply:
I considered that, but it is a zero byte file so net impact to the repo of sharing is zero (or might even be more depending on the length of the property name).  If we find a lot of these floating around we could de-dup them to remove clutter, but I felt this was better.